### ISSUE TITLE
[Minor UI issue] Provide system tray icon with transparend background

### ISSUE THREAD

**Issue by RequestPrivacy** [bisq-network/bisq2#1813]
bisq2 v2.0.1
Ubuntu flavor (Kubuntu 22.04 LTS)

Currently the system tray icon has a white background which looks not well integrated into the desktop if e.g. used in a dark mode environment. A fix would be to use an icon which has a transparent one so that the underlying color can shine through.

![image](https://github.com/bisq-network/bisq2/assets/72392544/c7cf6c64-0357-45e7-a911-0a8691040663)

**Comment by qxotk** [comment]
What is required?
Assuming I should get a dev environment set up, then find the icon in question, add a second version with appropriate name so UI devs can choose the version they want (transparent bg or not) ... then update the one used for system tray to be the transparent background one.

Does that sound right?

**Comment by HenrikJannsen** [comment]
Yes, its mainly to figure out which icon is used there. As far I am aware we only have transparent images, so might be needed to dig deeper to find out why the image is maybe converted is bitmap or the like. Maybe we need another format instead of png like svg. Feel free to give it a try. The other devs are all busy with other tasks.

**Comment by zabilal** [comment]
@HenrikJannsen Assign this to me

**Comment by puchis0323** [comment]
Can I take?
